ViAn/GUI/Analysis/analysiswidget.cpp

```diff
@@ -94,17 +94,25 @@ void AnalysisWidget::abort_analysis() {
     *abort = true;
 }
 
+void AnalysisWidget::abort_all_analysis() {
+    abort_all = true;
+    abort_analysis();
+}
+
 void AnalysisWidget::on_analysis_aborted() {
     analysis_queue.pop_front();
     delete current_analysis_item; // Delete item from tree
-    auto it = abort_map.find(current_method);   
+    auto it = abort_map.find(current_method);
```

Comment:
Should only erase the `current_method` is actually found otherwise this could lead to problems 
`if (it != abort_map.end())`